pkg/controllers/configobservation/oauth/template_conversions.go

```diff
@@ -82,7 +82,7 @@ func convertTemplatesWithBranding(cmLister corelistersv1.ConfigMapLister, config
 }
 
 func getConsoleBranding(cmLister corelistersv1.ConfigMapLister) (string, error) {
-	cm, err := cmLister.ConfigMaps("openshift-authentication").Get("v4-0-config-system-console-config")
+	cm, err := cmLister.ConfigMaps("openshift-config-managed").Get("wat")
```

Comment:
![giphy](https://user-images.githubusercontent.com/44136/86888368-5645e680-c0fa-11ea-856d-9a31f7ce1bba.gif)

Reply:
/hold
this will always result in "Not found", "openshift-config-managed" is not in namespaces that are being synced for the underlying informer

Reply:
> edit: not mentioning the wat

haha.. I have no idea how this happened. It was supposed to `console-config`. 

> this will always result in "Not found", "openshift-config-managed" is not in namespaces that are being synced for the underlying informer

Also, I will check on adding `openshift-config-managed` to list of informer namespaces.